tests/foreman/api/test_contentview.py

```diff
@@ -428,6 +428,48 @@ def test_positive_promote_3(self):
         self.assertEqual(len(cvv_attrs['environments']), REPEAT + 1)
         self.assertEqual(len(cvv_attrs['puppet_modules']), 1)
 
+    def test_add_normal_cv_to_composite(self):
+        """@Test: Create normal content view, publish and
+        add it to a new composite content view
+
+        @Assert: Content view can be created and assigned to
+        composite one through content view versions
+        mechanism
+
+        @Feature: ContentView
+
+        """
+        content_view = entities.ContentView(organization=self.org.id)
+        content_view.id = content_view.create_json()['id']
+        content_view.set_repository_ids([self.yum_repo.id])
+        content_view.publish()
+        for cvv_id in (
+                version['id']
+                for version
+                in content_view.read_json()['versions']):
+            entities.ContentViewVersion(id=cvv_id)
+
+        composite_cv = entities.ContentView(
+            id=entities.ContentView(
+                organization=self.org.id,
+                composite=True,
+                ).create_json()['id']
+        )
+
+        client.put(
+            composite_cv.path(),
+            {'content_view': {'component_ids': [cvv_id]}},
```

Comment:
Earlier in this test, I see that `cvv_id` is being used as the assigned-to variable in a loop. How many content view IDs are there? Does it matter which content view ID is usd in this PUT request, and are you sure that the loop will iterate through available values and assign the right value to `cvv_id`?